lib/adapters/mongodb/converters.js

```diff
@@ -17,11 +18,35 @@ module.exports = function() {
         }
     
         function toJsonApiSingle(resource) {
-            var mapped = _.mapKeys(resource, function (val, key) {
-                if (key === '_id') return 'id'
-                else return key
-            });
-            return _.omit(mapped, '__v')
+            return _.chain(resource)
+                .thru(_idToId)
+                .thru(createResourceLinks)
```

Comment:
Take this function out and place it in ```onPreResponse``` in ```plugin.js``` perhaps right after the includes: https://github.com/agco/hapi-harvester/blob/feature/AGCMD-41/lib/plugin.js#L181